Sources/StreamChatSwiftUI/ChatChannelList/DefaultChannelActions.swift

```diff
@@ -201,6 +206,20 @@ extension ChannelAction {
         return leaveConversation
     }
     
+    private static func viewInfo(for channel: ChatChannel) -> ChannelAction {
```

Comment:
Maybe write a test to verify the existence of this option?

Reply:
This is private so when I tried it cannot retrieve such option.